src/tools/devtools/memory.md

```diff
@@ -46,6 +46,17 @@ is no longer used (see [Dart garbage collection][]).
 
 [Dart garbage collection]: {{site.medium}}/flutter/flutter-dont-fear-the-garbage-collector-d69b3ff1ca30
 
+### Object types
+
+#### Disposable object
+
+A Dart object, that has method `dispose()`. The method, by convention, is expected to be invoked
+when the object isn't needed anymore to avoid memory leaks.
```

Comment:
```suggestion
A disposable object is any Dart object that defines a `dispose()` method.
To avoid memory leaks, Flutter expects that `dispose` is invoked when the
object isn't needed anymore.
```